### ISSUE TITLE
Add system property to enable/disable CallbackThreadInitializers

### ISSUE THREAD

**Issue by vinicius-tona** [gstreamer-java/gst1-java-core#147]
`GstBus` thread is leaking on macOS: https://groups.google.com/d/msg/gstreamer-java/zpcbAND9wnQ/8VVxgca6GgAJ

The initial idea is to implement a system property to switch off the CallbackThreadInitializers. 
Then we can look if it's a good idea to use Native.detach() and/or discuss upstream.

**Comment by neilcsmith-net** [comment]
This is done - eg. `-Dglib.detachCallbackThreads=true`
